app/controllers/application_controller.rb

```diff
@@ -2,4 +2,16 @@
 
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  layout :layout_by_resource
```

Comment:
@Mons-web I am sure we need to move this to `Web::ModuleController` since this also affects admin module.